script/support/sample_move.rb

```diff
@@ -0,0 +1,296 @@
+# frozen_string_literal: true
+
+class StudySample < ApplicationRecord
+end
+
+# added sample class as comments are not commentable at present 21/3/2019
+class Sample
+  include Commentable
+end
+
+class Asset
+  has_many :submitted_assets
+  has_many :orders, through: :submitted_assets, as: :asset
+end
```

Comment:
Asset shouldn't be used any more, it's only still around because I haven't had a change to kill it completely.